tpch/execute/q19.py

```diff
@@ -0,0 +1,17 @@
+from queries import q19
+
+from . import IO_FUNCS
+from . import lineitem
+from . import part
+
+fn = IO_FUNCS["pandas"]
+print(q19.query(fn(lineitem), fn(part)))
```

Comment:
gentle @MarcoGorelli, what do you think of the new version

Reply:
love this, great idea!